components/tools/Filters.js

```diff
@@ -82,7 +91,19 @@ export default function Filters({setOpenFilter}) {
       </div>
       <hr className="my-4" />
       <div className="flex flex-col gap-2 mx-4">
-        <div className="text-sm text-gray-500">LANGUAGE</div>
+        <div className="text-sm text-gray-500">
+          LANGUAGE
+          <span className="group relative">
+                {lang && <span className="w-48 text-xs border z-10 bg-white border-gray-200 shadow-md -left-2/3 absolute translate-x-1/3 -top-4 rounded px-2 py-1">
+                  {Data.properties.filters.properties.language.description.length >= 130 ?
+                    <div>{!readMore ? Data.properties.filters.properties.language.description.slice(0, 118) + "... " : Data.properties.filters.properties.language.description}<button className='cursor-pointer text-cyan-600' onClick={() => { setReadMore(!readMore) }} >{readMore ? " Show Less" : " Show More"}</button></div> :
+                    Data.properties.filters.properties.language.description}
+                </span>}
+                <button onClick={() => { setLang(!lang); setCategory(false); setTech(false) }} className="mx-1" ><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 translate-y-2">
```

Comment:
![image](https://user-images.githubusercontent.com/76521428/216063776-f809371c-6a6c-4eb0-a6b1-4d6430be7ba6.png)

Tooltip icon is perfect in terms of color and elements. Please don't add the SVG code directly into the file. Instead create a separate SVG component for the icon in `/components/icons` and import it as a component. Also, if possible, kindly reduce the size of the icon like width around `1.3 rem` and height as the initial of the SVG icon.